boards/cc2538dk/Makefile.include

```diff
@@ -7,7 +7,7 @@ PROGRAMMER_SERIAL ?= 06EB
 
 # setup serial terminal
 # the debug UART is always the second tty with the matching serial number:
-PORT_LINUX ?= $(word 2,$(shell $(RIOTTOOLS)/usb-serial/find-tty.sh '^$(PROGRAMMER_SERIAL)'))
+PORT_LINUX ?= $(word 1,$(shell $(RIOTTOOLS)/usb-serial/find-tty.sh '^$(PROGRAMMER_SERIAL)'))
```

Comment:
Why was this changed? Shouldn't it always be the second USB?

Reply:
Yes, when I changed it to 1, it always picked up the second USB. When it was 2, it didn't pick up any USB